shodan/cli/helpers.py

```diff
@@ -64,9 +64,26 @@ def get_banner_field(banner, flat_field):
     return None
 
 
+def filter_with_netmask(banner, netmask):
+    # filtering based on netmask is a more abstract concept than
+    # a mere check for a specific field and thus needs its own mechanism
+    # this will enable users to use the net:10.0.0.0/8 syntax they are used to
+    # to find specific networks from a big shodan download.
+    from ipaddress import ip_network, ip_address
```

Comment:
Please put import statements at the top of the file so it's easier to identify which modules are used in a file. This project is trying to follow the PEP8 guidelines:

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Roger wilco. Was trying to minimize impact but that obviously is more obfuscated.